splinterd/Cargo.toml

```diff
@@ -45,15 +45,20 @@ default = []
 stable = ["default"]
 
 experimental = [
+    "biome",
+    "biome-credentials",
     "circuit-read",
     "config-builder",
     "config-toml",
     "health",
 ]
 
-circuit-read = ["splinter/circuit-read"]
+biome = ["splinter/biome", "database"]
+biome-credentials = ["splinter/biome-credentials", "biome"]
 config-builder = []
 config-toml = ["config-builder"]
+circuit-read = ["splinter/circuit-read"]
```

Comment:
Looks like this was moved - it should be before `config-*`

Reply:
rebase issue. Fixed